### DISCUSSION TITLE
How can I solve this output garbled code when I use : system("pause");

### DISCUSSION THREAD

**Discussion by ZhgDgE** [cpeditor/cpeditor#1078] [Q&A]
![在这里插入图片描述](https://img-blog.csdnimg.cn/934a1a8a62114fcdb7eb1090628d67e8.png?x-oss-process=image/watermark,type_d3F5LXplbmhlaQ,shadow_50,text_Q1NETiBAWmhnRGdF,size_20,color_FFFFFF,t_70,g_se,x_16#pic_center)

![在这里插入图片描述](https://img-blog.csdnimg.cn/144aa24ac4ae48f0a6c953d4be90c04a.png?x-oss-process=image/watermark,type_d3F5LXplbmhlaQ,shadow_50,text_Q1NETiBAWmhnRGdF,size_20,color_FFFFFF,t_70,g_se,x_16#pic_center)

**Comment by ouuan** [comment]
Why do you use `system("pause")`?

**Comment by ShizuhaAki** [comment]
system pause is only needed for dev cpp 4.9.9.2 or previous versions, that closes the window immediately after program execution finishes. This is not needed for cpeditor. If you need portability for your template, you can add a macro -DCPEDITOR or something and remove the system pause here.

I suggest marking as notabug or won’t fix

On Apr 17, 2022, at 19:04, Yufan You ***@***.***> wrote:

﻿

Why do you use system("pause")?
